pkg/parser/sysl.wbnf

```diff
@@ -38,7 +38,13 @@ type_decl -> table | facade | alias | union;
 
 table -> mode=("!table"|"!type") NAME attribs?  ":" COMMENT*
```

Comment:
according to the antlr grammar, this can have a `SHORTCUT` after the colon. should you be adding this? do we need a test case for it?

Reply:
right, will add this. As for a test case, I'll see if there's any before making a new one.

Reply:
added the inline `SHORTCUT` and the test case